site/content/docs/5.0/components/tooltips.md

```diff
@@ -145,7 +145,7 @@ Elements with the `disabled` attribute aren't interactive, meaning users cannot
 Options can be passed via data attributes or JavaScript. For data attributes, append the option name to `data-`, as in `data-animation=""`.
 
 {{< callout warning >}}
-Note that for security reasons the `sanitize`, `sanitizeFn` and `whiteList` options cannot be supplied using data attributes.
+Note that for security reasons the `sanitize`, `sanitizeFn` and `allowList` options cannot be supplied using data attributes.
```

Comment:
Should this be turned into a partial?

```suggestion
Note that for security reasons the `sanitize`, `sanitizeFn`, and `allowList` options cannot be supplied using data attributes.
```

Reply:
not sure about partial ... it's (currently) only used twice ... would that be too much modularisation? no strong feelings about it either way though...let me know, happy to rip this out into partial